packages/arb-bridge-eth/test/sequencerInbox.spec.ts

```diff
@@ -0,0 +1,447 @@
+/*
+ * Copyright 2019-2020, Offchain Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-env node, mocha */
+
+import { ethers } from 'hardhat'
+import { BigNumber } from '@ethersproject/bignumber'
+import { TransactionReceipt } from '@ethersproject/providers'
+import { expect } from 'chai'
+import {
+  Bridge,
+  Bridge__factory,
+  Inbox,
+  MessageTester,
+  SequencerInbox,
+} from '../build/types'
+import { initializeAccounts } from './utils'
+import { Event } from '@ethersproject/contracts'
+import { Interface } from '@ethersproject/abi'
+import { BridgeInterface, MessageDeliveredEvent } from '../build/types/Bridge'
+import { Signer } from 'ethers'
+
+const mineBlocks = async (signer: Signer, count: number) => {
+  for (let index = 0; index < count; index++) {
+    await (
+      await signer.sendTransaction({
```

Comment:
I was checking out if we use [`provider.send()`](https://docs.ethers.io/v5/api/providers/jsonrpc-provider/#JsonRpcProvider-send) to mint blocks with [`evm_mine`](https://hardhat.org/hardhat-network/reference/#evm-mine) instead.
Seems like just as slow for large number of blocks, but the cool thing is that it lets us decide the timestamp of the next block.
We could use that to test the diff ways that block num vs timestamp can stray and how the inbox reacts to that

Reply:
updated to use `evm_mine`